src/main/java/com/salesforce/mirus/KafkaMonitor.java

```diff
@@ -89,6 +96,51 @@
         config.getEnablePartitionMatching()
             ? SourcePartitionValidator.MatchingStrategy.PARTITION
             : SourcePartitionValidator.MatchingStrategy.TOPIC;
+    this.routers = this.validateTransformations(config.transformations());
+    this.topicNamePrefix = config.getDestinationTopicNamePrefix();
+    this.topicNameSuffix = config.getDestinationTopicNameSuffix();
+  }
+
+  private List<Transformation<SourceRecord>> validateTransformations(
+      List<Transformation<SourceRecord>> transformations) {
+    List<Transformation<SourceRecord>> regexRouters = new ArrayList<>();
+    for (Transformation<SourceRecord> transform : transformations) {
+      String transformName = transform.getClass().getName();
+      if (transform instanceof RegexRouter) {
+        regexRouters.add(transform);
+        // Slightly awkward check to see if any other routing transforms are configured
+      } else if (transform.getClass().getName().contains("Router")) {
+        logger.error(
+            "Unsupported Router Transform %s found. Destination topic checking may not work.",
+            transformName);
```

Comment:
Should there be a way to disable destination topic checking in this case?

Reply:
Yeah, I think that's what I'm getting at with my earlier comment.

Reply:
That could be useful. Do you think it's worth limiting it to just this case ("Topics might be rerouted and we're not sure where, so always try to push them") or just having a global switch to disable destination checking? @pdavidson100 ?

Reply:
I think it would be safer to automatically disable in this case in combination with a clear warning, something like "WARN Destination topic validation will be disabled while unsupported router transform "%s" is enabled. Please ensure all possible destination topics exist."

Reply:
Reconsidering,  might be even safer if we made destination checking configurable per connector and then make the connector hard fail in this situation with the message "Unsupported configuration, please disable destination checking in connector configuration".  That way we fail fast.  Thoughts?

Reply:
I like the fail fast best, although it may break some existing configurations. Technically Mirus is still pre-1.0, do we have a versioning/breaking changes plan?

Reply:
Since we're still pre-1.0 I think we are OK to introduce this change, especially since we would likely just be surfacing an existing issue (and the fix is a simple config update called out in the error message). 

We absolutely need to get serious about issuing release notes covering important differences like this. I think we should add some release notes for 0.2 (the next feature release) to  this PR.